data-raw/fas_example_data.R

```diff
@@ -0,0 +1,198 @@
+  ) %>%
+  mutate(
+    percent_denominator = if_else(
+      value_type == 'percent',
+      gsub('^Percent of | ((that are)|(detected as)) .+$', '', endpoint),
+      NA_character_
+    ),
+    percent_denominator = case_match(
+      percent_denominator,
+      'GT8++IgG+ B cells' ~ 'GT8++ IgG+ B cells',
+      "epitope-specific (KO-GT8++) sequenced IgG BCRs" ~
+        "sequenced epitope-specific (KO-GT8++) IgG+ B cells",
+      .default = percent_denominator
+    ),
+    bcell_population = case_match(
+      bcell_population,
+      "epitope-specific (KO-GT8++) sequenced IgG BCRs that are VRC01-class" ~
+        'VRC01-class',
+      "KO-" ~ 'GT8++KO-',
+      "IgD-IgG+ B cells" ~ 'IgG+ B cells',
+      "IgD-IgG+ B cells that are GT8++ (without regard to KO binding status)" ~ 'GT8++ IgG+ B cells',
+      "epitope-specific (KO-GT8++) IgG+ B cells that have BCR heavy and light chains sequenced" ~
+        "sequenced epitope-specific (KO-GT8++) IgG+ B cells",
+      "epitope-specific (KO-GT8++) sequenced IgG BCRs that are not VRC01-class but have a 5-aa CDRL3" ~ "sequenced epitope-specific (KO-GT8++) IgG+ B cells that are not VRC01-class but have a 5-aa CDRL3",
+      "epitope-specific (KO-GT8++) sequenced IgG BCRs that are not VRC01-class but have a VH1-2 heavy chain" ~ "sequenced epitope-specific (KO-GT8++) IgG+ B cells that are not VRC01-class but have a VH1-2 heavy chain",
+      .default = gsub(' \\(without regard to KO binding status\\)|IgD\\-', '', bcell_population)
+    ),
+    igx_type = if_else(grepl('IgG', endpoint), 'IgG+', NA_character_),
+    Group = case_match(
+      Treatment,
+      '20 µg eOD-GT8 60mer + AS01B' ~ 1L,
+      '100 µg eOD-GT8 60mer + AS01B' ~ 2L
+    ),
+    dose = case_match(
+      Treatment,
+      '20 µg eOD-GT8 60mer + AS01B' ~ 20,
+      '100 µg eOD-GT8 60mer + AS01B' ~ 100
+    ),
+    dose_unit = case_match(
+      Treatment,
+      '20 µg eOD-GT8 60mer + AS01B' ~ 'µg',
+      '100 µg eOD-GT8 60mer + AS01B' ~ 'µg'
+    ),
+    visitno = sub('^V', '', Visit),
+    visit_units = 'weeks',
+    antigen_specificity = if_else(grepl('GT8[+][+]|VRC01[-]class', endpoint), 'GT8++', NA_character_),
+    epitope_specificity = if_else(grepl('KO[-]|VRC01[-]class', endpoint), 'KO-', NA_character_),
+    bnab_class = if_else(grepl('^VRC01[-]class$', bcell_population), 'VRC01-class', NA_character_),
+    source_assay = case_when(
+      value_type == 'count' & grepl('VRC01[-]class|sequenced', endpoint) ~ 'sequencing',
+      endpoint == 'Percent of epitope-specific (KO-GT8++) sequenced IgG BCRs that are VRC01-class' ~ 'sequencing',
+      value_type == 'percent' & grepl('VRC01[-]class|sequenced', endpoint) ~ 'flow and sequencing',
+      .default =  'flow'
+    ),
+    flag_bound = NA,
+    probeset = 'G001 PBMC (KO11 eOD-GT8)',
+    sample_type = 'PBMC',
+    source_file = 'https://github.com/SchiefLab/G001/raw/main/data/figures/flow_summary/flow_and_sequences.csv.gz',
+    PubID = sub('^PubID_', '', PubID)
+  ) %>%
```

Comment:
```suggestion
  ) %>%
  mutate(
    bcell_population = if_else((value_type == "percent") & (source_assay == "flow"), paste(bcell_population, percent_denominator), bcell_population),
    bcell_population = sub("GT8\\+\\+KO- GT8\\+\\+", "GT8++KO-", bcell_population)
  ) %>%
```

Requesting this change in order to streamline the values in the bcell_population column.

Before:
<img width="842" alt="Screenshot 2025-05-22 at 11 14 29 AM" src="https://github.com/user-attachments/assets/fc0d44af-9294-4121-9ef6-231b3e0f80a7" />

After:
<img width="1185" alt="Screenshot 2025-05-22 at 11 14 50 AM" src="https://github.com/user-attachments/assets/15bc5a03-762a-4991-8766-f777819de48f" />

Reply:
for example, distinguish "GT8++" as either "GT8++ B cells" or "GT8++ IgG+ B cells"